main/res/values/strings.xml

```diff
@@ -827,7 +827,9 @@
     <string name="init_bigSmileysOnMap">Big icons</string>
     <string name="init_summary_bigSmileysOnMap">If enabled, symbols like the \'found smiley\' or the \'corrected coordinates\' marker will be shown enlarged on the map instead of the cache type icon (requires restart).</string>
     <string name="init_global_wp_extraction_disable">Disable waypoint extraction</string>
-    <string name="init_summary_global_wp_extraction_disable">Globally disable waypoint extraction from personal cache notes. It can still be enabled for individual caches by deactivating the checkbox below the personal cache note</string>
+    <string name="init_summary_global_wp_extraction_disable">Globally disable waypoint extraction from personal cache notes. It can still be enabled for individual caches by deactivating the checkbox below the personal cache note.</string>
+    <string name="init_prevent_cache_note_merge">Prevent merging of Personal Cache Note.</string>
```

Comment:
The heading probably shouldn't contain a `.` at the end. Also it seems quite long. Maybe "Overwrite Personal Note"

Reply:
Removing the "." makes sense. The "quite long" I am not sure I understand what you are suggesting. Are you suggesting to replace the whole string by literally just "Overwrite Personal Note"? All three words in uppercase then?

Reply:
Check it out ....